src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java

```diff
@@ -293,6 +293,44 @@ public void testGetUser_successful() throws Exception {
         Mockito.verify(mockApiClient).get("https://api.sandbox.hyperwallet.com/rest/v4/users/test-user-token", userResponse.getClass());
     }
 
+    @Test
+    public void getUser_WithVerificationDocumentAndRejectReasons() throws Exception {
+        Hyperwallet client = new Hyperwallet("test-username", "test-password");
+        HyperwalletApiClient mockApiClient = createAndInjectHyperwalletApiClientMock(client);
+        HyperwalletUser hyperwalletUser = getHyperwalletUser();
+        final String url = "https://api.sandbox.hyperwallet.com/rest/v4/users/" + "test-userToken";
+        Mockito.when(mockApiClient.get(Mockito.eq(url), Mockito.eq(HyperwalletUser.class))).thenReturn(hyperwalletUser);
+
+        HyperwalletUser user = client.getUser("test-userToken");
+
+        Mockito.verify(mockApiClient).get(Mockito.eq(url), Mockito.eq(HyperwalletUser.class));
+        assertThat(user.getToken(), is(hyperwalletUser.getToken()));
+        assertThat(user.getDocuments().size(), is(1));
+        assertThat(user.getDocuments().get(0).getStatus(), is("INVALID"));
+        assertThat(user.getDocuments().get(0).getReasons().size(), is(1));
+        assertThat(user.getDocuments().get(0).getReasons().get(0).getName(), is(DocumentVerificationReason.DOCUMENT_EXPIRED));
+        assertThat(user.getDocuments().get(0).getReasons().get(0).getDescription(), is("Document has expired"));
```

Comment:
where are we setting the description? 
text is not matching with the value in enum.

Reply:
we should use enum directly for assert like:
is(DocumentVerificationReason.DOCUMENT_EXPIRED.getDescription())

Reply:
Have removed the description field, as we copy the name and description fields directly from api response, i think it doesn't need to come from enum description, mapping logic between name and description already exists in hw2 code.